package.json

```diff
@@ -4,7 +4,7 @@
   "description": "A 7dtd server manager made with Sails.js",
   "keywords": [],
   "dependencies": {
-    "7daystodie-api-wrapper": "^1.5.0",
+    "7daystodie-api-wrapper": "https://files.odin.gavinmogan.com/7daystodie-api-wrapper-1.5.1.tgz",
```

Comment:
Can this be changed back to the real dependency?

Reply:
Omg yea sorry

Just in case can you do another release of it? There's probably a reason I was using the patched version